executeCommand.py

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env python
+import subprocess
+
+
+serverList = ["artem@192.168.0.3:22", "artem@192.168.0.3:223"]
+command = "ls"
+
+for server in serverList:
+    serverparams=server.split(":")
+    o = subprocess.check_output(['ssh', '-p %s' %serverparams[1], '%s' %serverparams[0], '%s' %command], stderr=subprocess.STDOUT)
+    print(o)
```

Comment:
> ... collect output from all nodes. 
> Script should print collected output from all nodes on stdout, w/o using temp files.

1. Stdout output from executed command could be very large: imaging we run some long-running task which generates GB of data to stdout. Your main process (`executeCommand.py`) could be killed by OOM.

2. In case you run multiple commands in parallel, the output from different workers (subprocesses) will be mixed. How will you recognize which message to belongs which remote machine?

3. Exit-code is not checked

4. STDERR output is not captured/handled too